extensions/vscode/src/views/homeView.ts

```diff
@@ -443,13 +440,10 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
 
   private getActiveConfig(): Configuration | undefined {
     const savedState = this.getSelectionState();
-    if (!savedState || !savedState.configurationName) {
+    if (!savedState) {
       return undefined;
     }
-    return this.getConfigByName(
-      savedState.configurationName,
-      savedState.projectDir,
-    );
+    return this.getConfigBySelector(savedState);
```

Comment:
Things became a lot simpler once we were able to switch to the new strategy of the simple selector for the contentRecord, which then determined the active config.